commands/pro/pro.js

```diff
@@ -45,25 +40,30 @@ function loadAndMatchPros(bot) {
   updateProsMatcher();
   const helpChannel = bot.client.channels.find('name', 'helpdirectory');
 
-  return helpChannel.fetchMessages({limit: 100})
-  .then(function(messages) {
-    messages.forEach(function(messageObj) {
-      let match;
-      proLangRe.reset();
-      while (match = proLangRe.exec(messageObj.content))
+  return helpChannel.fetchMessages({ limit: 100 })
+  .then((messages) => {
+    messages.forEach((messageObj) => {
+      proLangRe.lastIndex = 0;
+      while (true) {
+        const match = proLangRe.exec(messageObj.content);
+        if (!match) {
+          break;
```

Comment:
this changed because of [no-cond-assign](http://eslint.org/docs/rules/no-cond-assign)
we could switch to `except-parens` mode instead